### PR TITLE
started task 7 , unsure if doing the right thing

### PR DESCRIPTION
**Pull request by hedieh623**

### CODE REVIEW

File: index.html
Comment (Rayhaan2k9) [inline index.html]: So this is the way to do buttons if you weren't using a framework like React. As you are using React, you don't need to do this as you can implement the feature using React!

File: script.js
Comment (Rayhaan2k9) [inline script.js]: Same with this

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
Hi Hedieh! It looks like you are trying to implement a voting feature the way you would if you were just using plain html and javascript. You don't need to do this as you are using React! I would recommend watching Chipie's lecture today as she is covering this very feature. It will also be recorded if you miss it. If you need any help after watching it, please send an nchelp 🙂

**Review by Augs0 — COMMENTED** [review]
Hi Hedieh, there's a small mistake in your voting logic which means I press downvote the number of votes actually goes up. Please fix this before continuing. Additionally, it looks like you merged this branch and then came back to it, so be sure to merge this branch again once it's all complete and approved.

**Comment by hedieh623** [comment]
Hi August and Rayhaan , thank you so much for your feedback :) I've just looked at it again and I'm not sure where the error is referring to on my code, as in on which line. Whats being shown on my app on the  browser for me is that when i click the up button it increments the count and when i click the down button it reduces the count :thinking_face:

**Pull request merged**
